include/TPP/Dialect/Xsmm/XsmmUtils.h

```diff
@@ -0,0 +1,27 @@
+//===- XsmmUtils.h - --------------------------------------------*- C++ -*-===//
+//
+// This file is licensed under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+#ifndef XSMM_DIALECT_XSMM_XSMMUTILS_H
+#define XSMM_DIALECT_XSMM_XSMMUTILS_H
+
+#include "TPP/Dialect/Xsmm/XsmmEnum.h"
+
+namespace mlir {
+class Type;
+class RewriterBase;
+
+namespace xsmm {
+namespace utils {
+
+xsmm::DataTypeAttr getXsmmDataType(RewriterBase &rewriter, Type type);
```

Comment:
Nit: If this is already in the `xsmm` namespace, does it really need to be called `getXsmmDataType`?